.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,13 @@
 ## Title
 
+<!-- Update the following example to be relevant -->
```

Comment:
No need for this. The list is the same as the skeleton project. Pull request must includes the checklists

Reply:
This is a public repo now, its good practice to make the process clear understandable to anyone opening a PR regardless of background knowledge.

Comment is only visible when creating / in edit view so it is harmless.

Reply:
What I mean by the list is not example, but people can add more if they want. We can update the comment to say minimum requirement :)

Reply:
Which list? What?

Yeah the language isn't clear around this is what I meant.
I've seen many open source PR templates include comments to clarify what things mean etc.

Reply:
What I mean is that this comment is about `Update the following example to be relevant`? What examples?

The next part is the list of items with checkboxes. So I assume you think these checkboxes are examples? They are checklist that have to be tickets before the PR is accepted for review (Same as what we have in projects PRs)

Reply:
The whole PR template is full of example content.
- Proposed Sniff name
- Disallowed Code Example
- Description
- Reaons
- Allowed Code Example
etc


I'll move the checklist though and give it a heading so that's clearer

Reply:
It is common sense to not keep the example. Move the comment to the top of the template so it is clear the comment is related to the entire thing. But I do think it is redundant to add this.